src/callbacks_stage/positivity_shallow_water.jl

```diff
@@ -0,0 +1,84 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+
+
+"""
+    PositivityPreservingLimiterShallowWater(; threshold, variables)
+
+The limiter is specifically designed for `ShallowWaterEquations`.
```

Comment:
You mean that it should include an `@ref` to either `AbstractShallowWaterEquations` or explicitly to `ShallowWaterEquations1D` and `ShallowWaterEquations2D`?

Reply:
Yes. If it is rendered as code, it should be real code - with a `@ref`. Otherwise, you can also just write
```
The limiter is specifically designed for the shallow water equations.
```
For example,
```suggestion
The limiter is specifically designed for the shallow water equations, i.e.,
the [`ShallowWaterEquations1D`](@ref) and the [`ShallowWaterEquations2D`](@ref).
```

Reply:
Done.